core/templates/domain/platform_feature/client-context.model.ts

```diff
@@ -17,31 +17,27 @@
  */
 
 export interface ClientContextBackendDict {
-  'client_type': string;
+  'platform_type': string;
   'browser_type': string;
-  'user_locale': string;
 }
 
 /**
  * Represents the client side information that is used to evaluate the value
  * of feature flags. This is used only in the frontend feature value retrieval.
  */
 export class ClientContext {
-  readonly clientType: string;
+  readonly platformType: string;
   readonly browserType: string;
-  readonly userLocale: string;
 
   constructor(
-      clientType: string, browserType: string, userLocale: string) {
-    this.clientType = clientType;
+      platformType: string, browserType: string) {
```

Comment:
Nit: move to line above since wrapping isn't needed anymore. Ditto below.

Reply:
Done.